packages/core/src/browser/style/search-box.css

```diff
@@ -0,0 +1,80 @@
+/********************************************************************************
```

Comment:
It should be a file move, not a copy from the `@theia/navigator` to the `@theia/core`.

Reply:
fixed